src/index.ts

```diff
@@ -223,7 +226,7 @@ const cocoDataAccess: DataAccessType = async (args: ArgsType): Promise<CocoDatas
   if (testPair.length > 0) {
     result.testLoader = testLoader;
   }
-
+  
```

Comment:
May I ask you to remove these spaces here

Reply:
ok,I have removed them, please check it.